src/cli_shared/cli/config.rs

```diff
@@ -55,6 +58,31 @@ impl Default for EventsConfig {
     }
 }
 
+/// Structure that defines `FEVM` configuration
+#[derive(Deserialize, Serialize, PartialEq, Eq, Debug, Clone)]
+#[cfg_attr(test, derive(derive_quickcheck_arbitrary::Arbitrary))]
+pub struct FevmConfig {
+    pub enable_eth_rpc: bool,
+    #[cfg_attr(test, arbitrary(gen(|g| u32::arbitrary(g) as _)))]
+    pub eth_trace_filter_max_results: usize,
+}
+
+impl Default for FevmConfig {
+    fn default() -> Self {
+        Self {
+            enable_eth_rpc: is_env_set_and_truthy(FOREST_FEVM_ENABLE_ETH_RPC).unwrap_or(false),
+            eth_trace_filter_max_results: 500,
+        }
+    }
+}
+
+#[derive(Deserialize, Serialize, PartialEq, Eq, Default, Debug, Clone)]
+#[cfg_attr(test, derive(derive_quickcheck_arbitrary::Arbitrary))]
+pub struct ChainIndexerConfig {
+    /// Number of retention epochs for indexed entries. Set to `None` to disable reclamation.
```

Comment:
```suggestion
    /// Number of retention epochs for indexed entries. Set to `None` to disable retention.
```
or maybe I don't understand _reclamation_ in this context?

Reply:
It means that after a certain number of retention epochs, the entries are garbage collected. If None is set, the entries will stay for ever.

Reply:
Should we use the term garbage collection here?